pkgs/racket-doc/scribblings/reference/cont.scrbl

```diff
@@ -94,12 +94,15 @@ handler, which accepts a single thunk to apply (with the prompt
 intact).}
 
 @defproc*[([(make-continuation-prompt-tag) continuation-prompt-tag?]
-           [(make-continuation-prompt-tag [sym symbol?]) continuation-prompt-tag?])]{
+           [(make-continuation-prompt-tag [name symbol?]) continuation-prompt-tag?])]{
 
 Creates a prompt tag that is not @racket[equal?] to the result of any
 other value (including prior or future results from
-@racket[make-continuation-prompt-tag]). The optional @racket[sym]
-argument, if supplied, is used when printing the prompt tag.}
+@racket[make-continuation-prompt-tag]). The optional @racket[name]
+argument, if supplied, specifies the name of the prompt tag.}
```

Comment:
How about adding "for printing or `object-name`"? Otherwise, it it may seem like the name is meant to have some other effect.

Reply:
Added.